src/pages/index.tsx

```diff
@@ -17,10 +18,13 @@ function Pages() {
 				<Route path="/confirm-sign-up">
 					<ConfirmSignUp />
 				</Route>
-				<Route path="/">
+				<Route exact path="/">
 					<GradeBook />
 				</Route>
 			</Switch>
+			<Switch>
+				<ErrorPage />
+			</Switch>
```

Comment:
I think the approach for this is to put this in a Router in the above switch instead of adding a switch. The current behaviour has the 404 page showing below the gradebook

Checkout the docs here: https://reactrouter.com/web/api/Switch

essentially this in the main switch
```
<Route>
    <ErrorPage />
</Route>
```